### PR TITLE
Upgrade swaggerapi/swagger-ui:v5.1.0

### PR DESCRIPTION
**Pull request by dszakallas**
Updates `nginx:1.23.2-alpine` to `nginx:1.25.1-alpine` which fixes security vulnerabilities including [one critical](https://hub.docker.com/layers/library/nginx/1.23.2-alpine/images/sha256-0f2ab24c6aba5d96fcf6e7a736333f26dca1acf5fa8def4c276f6efc7d56251f?context=explore).

Although major increase, release note point out that only backward-compatible changes were introduced to [swagger-ui:5.0.0](https://github.com/swagger-api/swagger-ui/releases/tag/v5.0.0).

### CODE REVIEW

**Comment by dszakallas** [comment]
Critical vulns to be fixed by #46 . Still might worth to upgrade, but we need changes in build workflow before merging this.

**Comment by dszakallas** [comment]
Removed patching of swagger-ui docker image as we will not need it anymore. Closes #46 as incompatible with the approach we decided to take.

File: Dockerfile [hunk 1]
```diff
@@ -15,12 +15,9 @@
 # To use ansible playbook: docker exec <CONTAINER_ID> ansible-playbook samples/ansible/<PATH_TO_PLAYBOOK>
 # To use pfctl: docker exec <CONTAINER_ID> pfctl <COMMAND>
 # You can also enter interactive session: docker exec -it <CONTAINER_ID> /bin/bash
-
-ARG LOCAL_REGISTRY=localhost:5000/ SWAGGER_UI_IMAGE=swagger-ui:v4.15.5
-
 ARG ANSIBLE_PLAYBOOKS_COMMIT=df67f78
 
-FROM ${LOCAL_REGISTRY}${SWAGGER_UI_IMAGE}
+FROM swaggerapi/swagger-ui:v5.1.0
```
Comment (dimleevik) [inline Dockerfile:20]: Why not just swagger-ui:latest?

Comment (dszakallas) [reply Dockerfile:20]: Using latest is not best practice, as they highly degrade build reproducibility. As a follow-up we should investigate introducing dependabot or another upgrade nagging bot.

**Review by dimleevik — APPROVED** [review]

**Pull request merged**